src/charm.py

```diff
@@ -93,7 +93,9 @@ def reconcile(self) -> None:
         )
         if self.state.custom_config_repository:
             wazuh.configure_git(
-                container, self.state.custom_config_repository, self.state.custom_config_ssh_key
+                container,
+                str(self.state.custom_config_repository),
```

Comment:
Where is the str splitting performed?
Is the `configure_git` a method from a library that expects a str as input?

Reply:
Inside that method, which is in another module

Reply:
Ok, that method is taking a str, so the str casting should be a good idea.